.github/ISSUE_TEMPLATE/release-plan.md

```diff
@@ -13,7 +13,9 @@ assignees: 'Guite, craigh'
 - [ ] Check if there are no open issues which are not labeled as `blocker` but seem critical
 - [ ] Ensure all VA modules and own components have been released to ensure all changes are correctly pulled.
 - [ ] Ensure all dependencies and `composer.lock` are up to date (execute `composer update`)
-- [ ] Ensure all translation templates are up to date (execute `php -dmemory_limit=2G bin/console translation:extract zikula en`)
+- [ ] Ensure all translation templates are up to date
+  - [ ] first execute `php -dmemory_limit=2G bin/console translation:extract zikula en`
+  - [ ] second execute `php -dmemory_limit=2G bin/console zikula:translation:keytovalue`
```

Comment:
this is done in the **action** right? it just isn't committed. maybe we could just commit it?

Reply:
I want to have this in a separate action, but this is currently blocked by https://github.com/php-translation/symfony-bundle/issues/404 because it would create a PR on every run.